myscript.js

```diff
@@ -1,29 +1,30 @@
 //list item template
-var itemTemplate = 
+var ITEM_TEMPLATE =
     '<a href="#" class="list-group-item"> \
         <input type="checkbox" onclick="completeItem(this)" value=""> \
         <span class="badge">A</span> \
-        <button class="delete-button" onclick="deleteItem(this)">Delete</button></a>';
+        <button class="delete-button" onclick="deleteItem(this)">Delete</button> \
+        <span class="item-text"></span></a>';
 
 //addItem function
-var addItem = function () {
-    var list=$("#notCompletedItems");
-    var newItem=$(itemTemplate); 
-    var userInput = document.getElementById("text-input").value;
-    //put text input into the innerHTML of the newItem
-    newItem.append(userInput);
-    list.append(newItem);
-    //TODO clear the text input box
-    document.getElementById("text-input").value = '';
+function addItem() {
+    var $list = $('#notCompletedItems');
+    var userInput = $('#text-input').val();
+
+    var $newItem = $(ITEM_TEMPLATE);
+
+    $newItem.find('.item-text').text(userInput);
```

Comment:
Here, instead of doing `newItem.append(userInput)` I'm using that new `<span>` I created to put the text there.

In the old way, an item would look like this:
```
<a href="#" class="list-group-item">
    <input type="checkbox" onclick="completeItem(this)" value="">
    <span class="badge">A</span>
    <button class="delete-button" onclick="deleteItem(this)">Delete</button>
    **Here's my thing to do**
</a>
```

In the new way, there's a span around the text, like: `<span class="item-text">**Here's my thing to do**</span>`. This just makes it so that later, if you want you can actually get the text back out. Plus you can style it with CSS with a `.item-text { color: red }` if you wanted